schduler/clock.py

```diff
@@ -0,0 +1,30 @@
+import os
+from rq import Queue
+from worker import conn
+import logging
+from apscheduler.schedulers.blocking import BlockingScheduler
+
+
+logger = logging.info(__name__)
+sched = BlockingScheduler()
+q = Queue(connection=conn)
+
+
+def trigger_updatedb():
+    os.system("python src/updatedb.py")
+
+
+def trigger_failures():
+    os.system("python tools/failures.py")
+
+
+@sched.scheduled_job('interval', hours=12)
+def timed_trigger_updatedb():
+    q.enqueue(trigger_updatedb())
+
+
+@sched.scheduled_job('interval', hours=6)
+def timed_trigger_failures():
+    q.enqueue(trigger_failures())
```

Comment:
@jmaher ,sorry for the late reply. I was a kind of lazy these days :P 

The 'hours' in here represent the hour between  runs. And yes you are right, these two tasks can run at the same time. Because the rq use 'worker' to execute task and it seems like rq will init a new worker for each task independently[1]. But, however, the blockingschdule we used in [2] will block other scheduler when it's running[3] and I haven't google out a solution out about the issue you mentioned above. So I propose to change the interval hours of failures.py run from **6** to **5** or **7**. I believe it can avoid the issue of two scripts been executed at the same time. :)

[1] http://python-rq.org/docs/workers/
[2] https://github.com/mozilla/ouija/pull/116/files#diff-d67473fe5745bd14193c0a193ef44dc0R5
[3] https://apscheduler.readthedocs.io/en/latest/userguide.html#choosing-the-right-scheduler-job-store-s-executor-s-and-trigger-s